overlay/usr/lib/inithooks/bin/invoice-ninja.py

```diff
@@ -62,7 +62,6 @@ def main():
     
     m = MySQL()
     m.execute('UPDATE ninja.users SET password=%s WHERE id=1;', (hashpass,))
-    m.execute('UPDATE ninja.users SET username=%s WHERE id=1;', (email,))
```

Comment:
Out of curiosity, does it just use the `admin` username now? (Previously it was set up to use the email as the username).

Reply:
Nah there isn't even a username anymore. Now there's a "first name" and "last name" field (set during curl install). Login is still performed with email though.